### PR TITLE
Update AGP from 8.0.0-alpha02 to 8.0.0-alpha03

### PR DESCRIPTION
**Pull request by erichaagdev**
This PR updates the supported version of AGP from `8.0.0-alpha02` to `8.0.0-alpha03`.

Note the requirement of `namespace` in the `android` extension.

### CODE REVIEW

File: src/test/groovy/org/gradle/android/SimpleAndroidApp.groovy [hunk 1]
```diff
@@ -71,16 +71,14 @@ class SimpleAndroidApp {
         writeActivity(library, libPackage, libraryActivity)
         writeRoomSourcesIfEnabled(library, libPackage)
         file("${library}/src/main/AndroidManifest.xml") << """<?xml version="1.0" encoding="utf-8"?>
-                <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-                    package="${libPackage}">
+                <manifest xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (cdsap) [inline src/test/groovy/org/gradle/android/SimpleAndroidApp.groovy:74]: is the attribute `package` totally removed/deprecated in `alpha03`?

**Review by cdsap — APPROVED** [review]

File: src/test/groovy/org/gradle/android/SimpleAndroidApp.groovy (hunk 1 above)
Comment (erichaagdev) [reply src/test/groovy/org/gradle/android/SimpleAndroidApp.groovy:74]: It appears so. See [this build scan](https://ge.solutions-team.gradle.com/s/kamnc2zxwalzu/tests/:testAndroid8_0_0_alpha03/org.gradle.android.RoomSchemaLocationWorkaroundTest/schemas%20are%20generated%20into%20task-specific%20directory%20and%20are%20cacheable%20with%20kotlin%20and%20kapt%20workers%20enabled%20(Android%208.0.0-alpha03)%20(Kotlin%201.5.32)?focused-exception-line=0-49&top-execution=1) where I left it in.

> Setting the namespace via the package attribute in the source AndroidManifest.xml is no longer supported, and the value is ignored.

Comment (runningcode) [reply src/test/groovy/org/gradle/android/SimpleAndroidApp.groovy:74]: nice, thanks for updating this!

**Review by runningcode — APPROVED** [review]
LGTM!

**Pull request merged**